features/ssl/README.md

```diff
@@ -0,0 +1,158 @@
+## SSL
+
+You can secure traffic between the driver and Cassandra with SSL. This
+section describes the driver-side configuration; it assumes that you've
+already
+[configured SSL in Cassandra](http://docs.datastax.com/en/cassandra/2.0/cassandra/security/secureSSLClientToNode_t.html).
+
+### Preparing the certificates
+
+#### Client truststore
+
+This allows the client to verify the identity of all the Cassandra nodes
+it connects to.
+
+As part of configuring Cassandra, you should already have
```

Comment:
I think it would be good to either change this sentence to also indicate that you could just trust the CA that signed the certificates on the nodes or reduce the terminology that each nodes certificate would be trusted.  I don't think we need to cover end to end how to configure the C* side of things (so just linking to the docs as done here is fine).

Unfortunately the datastax docs demonstrate creating a self-signed certificate on each node, but that does not work well as each node needs to be trusted individually and is hard to manage, especially in the client case (i.e. you'd have to reload your truststore every time you add a new node).   [These slides](http://www.slideshare.net/zznate/hardening-cassandra-for-compliance-or-paranoia/31) and accompanying [blog post](http://thelastpickle.com/blog/2015/09/30/hardening-cassandra-step-by-step-part-1-server-to-server.html) define a good approach for that (only focuses on internode encryption though).